lib/ship.rb

```diff
@@ -0,0 +1,17 @@
+class Ship
+  attr_accessor :name, :length, :health
+
+  def initialize(name,length)
+    @name = name
+    @length = length
+    @health = length
```

Comment:
It's a tad bit confusing that health and length are derived from the same parameter. Should we change the API to be more like `(name, length, health)`? 

We might consider having a starting health which length is aliased to, that way we track the more important concepts of `health` and `starting_health` through data, while length just coincidentally has `starting_health`'s value.

Reply:
We can't require health to be an argument when initializing per the interaction patterns provided. It does need to be an attribute of the object however, so its included there and set to the same as the length of the ship as explained in the instructions.